config/migrations/2023/lpdc/20230512160000-add-concept-display-configurations.sparql

```diff
@@ -0,0 +1,37 @@
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX cpsv: <http://purl.org/vocab/cpsv#>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX lpdcExt: <https://productencatalogus.data.vlaanderen.be/ns/ipdc-lpdc#>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX pav: <http://purl.org/pav/>
+
+INSERT {
+  GRAPH ?graph {
+    ?concept lpdcExt:hasConceptDisplayConfiguration ?configUri .
+    ?configUri a lpdcExt:ConceptDisplayConfiguration ;
+      mu:uuid ?configId ;
+      lpdcExt:conceptIsNew true ;
```

Comment:
Double checked with Stefanie. This needs to be set to false in order to reduce the noise this will otherwise cause. The risk is that users miss some new concepts, but it would have been the same if this feature wasn't implemented.